export/serializers/csv_serializer.py

```diff
@@ -21,14 +21,21 @@
 import codecs
 import csv
 import re
-import StringIO
+import six
+try:
+    import StringIO
+except ImportError:
+    import io as StringIO
 
 from itertools import groupby
 from operator import itemgetter
 
 from django.core.serializers.python import Serializer as PythonSerializer
 from django.core.serializers.python import Deserializer as PythonDeserializer
-from django.utils.encoding import smart_unicode
+try:
+    from django.utils.encoding import smart_unicode
```

Comment:
Going forward, this will be called `smart_text` and will be referred to as such in [the docs](https://docs.djangoproject.com/en/1.11/topics/python3/#string-handling)

I think it's preferable we use that name. Please change this accordingly here and where it is used:
```
try
    from django.utils.encoding import smart_unicode as smart_text
except ImportError:
    from django.utils.encoding import smart_text